qualang_tools/plot/Fitting/fitting_functions.py

```diff
@@ -0,0 +1,348 @@
+from scipy import optimize
+import numpy as np
+
+
+def fit_ramsey(x, y):
+    """
+    Create a fit to Ramsey experiment of the form
+
+    .. math::
+    f(x) = uncertainty_population * (1 - np.exp(-x * (1/tau))) + amp / 2 * (
+        np.exp(-x * (1/tau))
+        * (initial_offset * 2 + np.cos(2 * np.pi * f * x + phase))
+        )
+
+    for unknown parameters `f`, `phase`, `tau`, `amp`, `uncertainty_population` and `initial_offset`.
+
+    :param x:  The data on the x axis
+    :param y: The data on the y axis
```

Comment:
It looks like this should be the transition probability to be in the excited state right ?

Reply:
So it is actually the transmission in a.u. no?

Reply:
What do you call transmission? Is it sqrt(I**2+Q**2)?

Reply:
Yes, it is sqrt(I+Q)

Reply:
I guess that if we renormalize the data internally to do the fitting it doesn't really matter, it can be I, Q, sqrt(I^2+Q^2), phase, transition probability...